db-postgresql/dataGenerator.js

```diff
@@ -0,0 +1,56 @@
+const faker = require('faker');
+const fs = require('fs');
+const file = fs.createWriteStream('./mockData.csv');
+
+const createCompanies = ( numOfChar ) => {
+  const options = ['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M', 'N', 'O', 'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'W', 'X', 'Y', 'Z'];
```

Comment:
I see no methods called to `options` that are unique to arrays in the `createCompanies` function, this can actually be changed to a string if the line length causes any inconvenience with page viewing